web/modules/custom/pineapple_general/pineapple_general.module

```diff
@@ -93,6 +93,16 @@ function pineapple_general_form_alter(&$form, \Drupal\Core\Form\FormStateInterfa
           }
         }
       }
+      if (key_exists('form_id', $form_state->getBuildInfo())) {
+        $build_info = $form_state->getBuildInfo();
+        if ($build_info['form_id'] == 'node_recipe_edit_form' || $build_info['form_id'] == 'node_recipe_form') {
```

Comment:
Why are you checking for the form Id again when you're already in a switch statement that's already checked for those specific form ids?

Reply:
Fair point, it was an oversight. Done.